.github/workflows/build.yml

```diff
@@ -9,95 +9,61 @@ name: Continuous integration
 
-      - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
+      - name: Setup nodejs
+        uses: actions/setup-node@v1
         with:
-          profile: minimal
-          toolchain: ${{ matrix.rust }}
-          override: true
-          components: rustfmt, clippy
+          node-version: '12.x'
 
-      - name: Cargo build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: --release --all
-      
-      - name: Upload Artifact (Linux/MacOS)
-        uses: actions/upload-artifact@v1
-        if: matrix.rust == 'stable' && matrix.os != 'windows-latest'
-        with:
-          name: juno-${{matrix.os}}
-          path: target/release/juno
-      
-      - name: Upload Artifact (Windows)
-        uses: actions/upload-artifact@v1
-        if: matrix.rust == 'stable' && matrix.os == 'windows-latest'
-        with:
-          name: juno-${{matrix.os}}
-          path: target/release/juno.exe
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Linting and Test
+        run: npm run test:prod
+
+      - name: Run Build
+        run: npm run build
 
-  release-master: # Publish release on push to master
+  # Publish release on push to master
+  release-master:
     if: github.ref == 'refs/heads/master'
     runs-on: ubuntu-latest
     needs: build
     steps:
       - uses: actions/checkout@v2
+
       - run: git fetch --all --tags
 
       - name: Check Release Version
         uses: thebongy/version-check@v1
         with:
-          file: Cargo.toml
+          file: package.json
           tagFormat: v${version}
         id: version_check
-      
-      - name: Download Windows Artifact
-        uses: actions/download-artifact@v1
-        with:
-          name: juno-windows-latest
-          path: release/windows
-
-      - name: Download MacOS Artifact
-        uses: actions/download-artifact@v1
-        with:
-          name: juno-macos-latest
-          path: release/macos
-
-      - name: Download Linux Artifact
-        uses: actions/download-artifact@v1
-        with:
-          name: juno-ubuntu-latest
-          path: release/linux
-      
-      - name: Rename Artifacts
-        run: |
-          mv release/windows/juno.exe release/windows/juno-${{steps.version_check.outputs.releaseVersion}}-windows.exe
-          mv release/macos/juno release/macos/juno-${{steps.version_check.outputs.releaseVersion}}-macos
-          mv release/linux/juno release/linux/juno-${{steps.version_check.outputs.releaseVersion}}-linux
 
       - name: Publish Release
-        uses: softprops/action-gh-release@v1
+        id: create_release
+        uses: actions/create-release@latest
         with:
-          files: release/**/*
-          tag_name: ${{steps.version_check.outputs.releaseVersion}}
+          tag_name: ${{ steps.version_check.outputs.releaseVersion }}
+          release_name: Juno-node @Linux Release ${{ steps.version_check.outputs.releaseVersion }}
```

Comment:
Why Linux release? The release would just be a npm `tar` file, it would be system independent. Just juno-node <version> release should be enough